src/main/java/com/sda/groupa/shippingcostcalculator/login/controller/AppController.java

```diff
@@ -35,22 +29,39 @@ public ModelAndView getLoginPage() {
     }
 
     @GetMapping("/")
-    public ModelAndView getHomePage(Principal principal, HttpServletRequest request) {
+    public String getHomePage() {
 
-        ModelAndView modelAndView = new ModelAndView("home");
+        UserAuthority userAuthority = userProvider.getUser().getRole().getUserAuthority();
+        String redirect;
 
-        String userName = principal.getName();
+        if(userAuthority.equals(UserAuthority.DRIVER)){
```

Comment:
Perfekcyjna strategia. `UserRedirectionStrategy`.

Reply:
Zostawię do późniejszego commita tak żebyśmy mogli iść dalej z kodem, ale zrobię to ; )

Reply:
Jednak dodane